aws/lambda/hud-query-proxy/lambda_function.py

```diff
@@ -0,0 +1,238 @@
+"""
+Query the database backing metrics.pytorch.org for recent commits and statuses,
+then push these to S3 at ossci-job-status/single/{branch}.tar.gz
+
+This fetches the 100 most recent commits for the branch (GitHub doesn't send
+push events for stacks of commits so we'd need to recover those manually), then
+gets their GitHub Actions job statuses and status events (CircleCI, CodeCov,
+Jenkins, etc.)
+
+Use environment variables 'num_commits' and 'branches' to configure the lambda
+"""
+import os
+import re
+import sys
+import gzip
+import json
+import io
+import boto3
+
+from typing import *
+
+import asyncio
+import aiomysql
+
+
+def eprint(*args):
+    print(*args, file=sys.stderr)
+
+
+async def run_query(query: str, params: List[str] = None) -> Dict[str, Any]:
+    eprint(f"Executing '{query}' with params: {params}")
```

Comment:
Isn't there a better way to write something to the log?

Reply:
sure we could use `logging` but it ends up the same thing anyways, cloudwatch auto-adds in things like timestamps so printing them here is redundant